okta/help.md

```diff
@@ -616,45 +616,95 @@ Example output:
 
 ```
 {
-  "id": "00ug5xak4uqrOrj8Q0h7",
-  "status": "STAGED",
-  "created": "2018-09-06T19:00:06.000Z",
-  "lastUpdated": "2018-09-06T19:00:06.000Z",
-  "profile": {
-    "firstName": "Isaac",
-    "lastName": "Brock",
-    "mobilePhone": "555-415-1337",
-    "login": "user@example.com",
-    "email": "user@example.com"
-  },
-  "credentials": {
-    "emails": [
-      {
-        "value": "user@example.com",
-        "status": "VERIFIED",
-        "type": "PRIMARY"
-      }
-    ],
-    "provider": {
-      "type": "OKTA",
-      "name": "OKTA"
-    }
-  },
-  "_links": {
-    "activate": {
-      "href": "https://dev-114295.oktapreview.com/api/v1/users/00ug5xak4uqrOrj8Q0h7/lifecycle/activate",
-      "method": "POST"
-    },
-    "self": {
-      "href": "https://dev-114295.oktapreview.com/api/v1/users/00ug5xak4uqrOrj8Q0h7"
-    }
-  }
+	"users_added_from_groups": [{
```

Comment:
Formatting here isn't right. Two space, indention. I think the original might be accurate and accidentally changed?

Reply:
Is this a regenerate bug?

Reply:
copy past error on my part

Reply:
copy past error on my part